Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.h

```diff
@@ -283,6 +283,10 @@ class WebsiteDataStore : public API::ObjectImpl<API::Object::Type::WebsiteDataSt
     void resetCacheMaxAgeCapForPrevalentResources(CompletionHandler<void()>&&);
     const WebsiteDataStoreConfiguration::Directories& resolvedDirectories() const;
     FileSystem::Salt mediaKeysStorageSalt() const;
+#if ENABLE(SCREEN_TIME)
+    void removeScreenTimeData(HashSet<URL> websitesToRemove);
```

Comment:
You probably want to pass an r-value reference or const l-value reference, whichever is more appropriate for callers? Or else you'd be copying the set on every invocation.